pygmt/src/rose.py

```diff
@@ -3,10 +3,17 @@
 """
 
 from pygmt.clib import Session
-from pygmt.helpers import build_arg_string, fmt_docstring, kwargs_to_strings, use_alias
+from pygmt.helpers import (
+    build_arg_string,
+    deprecate_parameter,
+    fmt_docstring,
+    kwargs_to_strings,
+    use_alias,
+)
 
 
 @fmt_docstring
+@deprecate_parameter("columns", "incols", "v0.4.0", remove_version="v0.6.0")
```

Comment:
@michaelgrund You also need to change `i="columns"` to `i="incols"` at line 32.

Reply:
Sure, thanks for mentioning @seisman 😉.